beacon-chain/core/epoch/precompute/reward_penalty.go

```diff
@@ -116,7 +116,7 @@ func attestationDelta(state *stateTrie.BeaconState, pBal *Balance, v *Validator)
 	finalityDelay := e - finalizedEpoch
 	if finalityDelay > params.BeaconConfig().MinEpochsToInactivityPenalty {
 		p += params.BeaconConfig().BaseRewardsPerEpoch * br
-		if !v.IsPrevEpochTargetAttester {
+		if !v.IsPrevEpochTargetAttester && !v.IsSlashed {
```

Comment:
Shouldn't this slashing check be inversed? Why would we punish validators that aren't slashed?

Reply:
This is inactive penalty

Reply:
You are right. It should be `|| v.IsSlashed` Thanks!